pool/store/mariadb_store.py

```diff
@@ -39,15 +39,17 @@ class MariadbPoolStore(AbstractPoolStore):
     Pool store based on MariaDB.
     """
     async def connect(self):
-        self.pool = await aiomysql.create_pool(
-        minsize=1, 
-        maxsize=12,
-        host=db_host,
-        port=db_port,
-        user=db_user,
-        password=db_password,
-        db=db_name )
-        
+        try:
+            self.pool = await aiomysql.create_pool(
+            minsize=1, 
+            maxsize=12,
+            host=db_host,
+            port=db_port,
+            user=db_user,
+            password=db_password,
+            db=db_name )
+        except Exception as e:
+                self.log.error(f"Error In Database Config. Check your config file! {e}")
```

Comment:
too many spaces, also log formatted in place while logger.error calls format on the first argument and *args, but only if it would actually emit something, so you should use
```py
self.log.error("Error In Database Config. Check your config file! %s", e)
```
to enable that lazy formatting, especially for non-error log statements.
Here, however, you are logging an exception, and logger.exception(message) is exactly for that - it shows a nice stacktrace too!

Reply:
Given lazy formatting as requested